pkg/controller/kibana/policy_config_setting.go

```diff
@@ -0,0 +1,65 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package kibana
+
+import (
+	"context"
+	"encoding/json"
+
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+
+	kibanav1 "github.com/elastic/cloud-on-k8s/v2/pkg/apis/kibana/v1"
+	common "github.com/elastic/cloud-on-k8s/v2/pkg/controller/common/settings"
+	"github.com/elastic/cloud-on-k8s/v2/pkg/controller/stackconfigpolicy"
+	"github.com/elastic/cloud-on-k8s/v2/pkg/utils/k8s"
+)
+
+// PolicyConfig is a structure for storing Kibana config from the StackConfigPolicy
+type PolicyConfig struct {
+	KibanaConfig      *common.CanonicalConfig
+	PolicyAnnotations map[string]string
```

Comment:
Nit: `policyConfig.PodAnnotations` sounds better to me.
```suggestion
	PodAnnotations map[string]string
```